intro-Linux/ch_uploading_files.tex

```diff
@@ -2,23 +2,25 @@ \chapter{Uploading/downloading/editing files}
 
 \subsection{Uploading/downloading files}
 
-Detailed information about uploading/downloading files with ``scp'' (using an OS
-X or Linux client) or WinSCP (Windows client), see
-\url{http://hpc.ugent.be/userwiki/index.php/User:VscCopy}.
+To transfer files from and to the HPC, see
+\href{https://hpcugent.github.io/vsc_user_docs/pdf/intro-HPC-linux-gent.pdf#section.3.2}{chapter 3.2 of the HPC manual}.
+
+\ifwindows
 
 After copying files it is advised to run
 
 \begin{prompt}
 %\shellcmd{dos2unix filename}
 \end{prompt}
 
-as this will fix any problems with windows / unix conversion.
+as this will fix any problems with Windows/Unix conversion.
 
+\fi
 \section{Symlinks for data/scratch}
 
 As we end up in the home directory when connecting, it would be convenient if we
-could access our data and vo storage. To facilitate this we shall create
-symlinks to them in our home directory.
+could access our data and VO storage. To facilitate this we shall create
+symlinks to them in our home directory. This will create 4 directories pointing to the respective storages
```

Comment:
Not directories, symbolic links. You can call them "shortcuts" or something like that, if you want to explain it a bit.

Maybe also mention that they (sort of) look like directories in WinSCP.